gtk3/sample/misc/print.rb

```diff
@@ -1,245 +1,246 @@
 #!/usr/bin/env ruby
 =begin
-  print.rb - Gtk::Print* sample
+   print.rb - Gtk::Print* sample
 
-  Copyright (C) 2007 Mathieu Blondel
-  This program is licenced under the same licence as Ruby-GNOME2.
-
-  $Id: print.rb,v 1.1 2007/08/05 15:41:31 mutoh Exp $
+   Copyright (C) 2007 Mathieu Blondel
+   Copyright (C) 2007-2015 Ruby-GNOME2 Project Team
+   This program is licenced under the same licence as Ruby-GNOME2.
 =end
 
-require 'gtk3'
+require "gtk3"
 
 # Values returned by a Pango::Layout or set in a Pango::Layout
 # are always multiple of Pango::SCALE. Those are convenience methods to
 # deal with that.
 class Pango::Layout
-    def size_in_points
-        self.size.collect { |v| v / Pango::SCALE }
-    end
+  def size_in_points
+    size.collect { |v| v / Pango::SCALE }
+  end
 
-    def width_in_points
-        self.size[0] / Pango::SCALE
-    end
+  def width_in_points
+    size[0] / Pango::SCALE
+  end
 
-    def height_in_points
-        self.size[1] / Pango::SCALE
-    end
+  def height_in_points
+    size[1] / Pango::SCALE
+  end
 
-    def width_in_points=(width)
-        self.width = width * Pango::SCALE
-    end
+  def width_in_points=(width)
+    width *= Pango::SCALE
```

Comment:
It enbugs. `self.width =` is needed. The code just assigns to the local variable.